suite/tests/api/ir_x86.c

```diff
@@ -2765,6 +2765,25 @@ test_extra_leading_prefixes(void *dc)
 #endif
 }
 
+static void
+test_ud1_operands(void *dc)
```

Comment:
It seems confusing to call it ud1 here yet ud2b everywhere else (including in the disasm strings inside this function): add a comment to the decode_table entry mentioning it is also called "ud1"?

Maybe also rename this function to use DR's name?

Reply:
What are the backwards-compat limitations here on renaming ud2b to ud1 and ud2a to ud2? Do we need to preserve #defines for the old OP_xxx, instr_create_xxx, etc?

Reply:
We should preserve the OP_ numbers (no adding/removing in the middle) and we should preserve the old OP_ud2b.  What I would do is rename the OP_ud2b in the enum (and table) to OP_ud1 and then add OP_ud2a as an alias define equal to OP_ud1.  I would add a release note to release.dox as a minor compat change that the instr_create and decoding of OP_ud2b is now different and would not preserve the exact semantics of the instr_create.

Reply:
Alright, how about this?

Reply:
Yes, looks good.

Reply:
Don't forget to update the PR description (which becomes the commit msg) to mention the renames.